Makefile

```diff
@@ -20,9 +20,8 @@ linux_compile:
 .PHONY: compile
```

Comment:
L18 probably should also from /artifacts/flyte-copilot  -> artifacts/flyte-copilot

Reply:
It went through the legacy stow configuration and set the access key and secret correctly.
```
 artifacts/flyte-copilot config discover    --storage.connection.endpoint localhost:30084 --storage.type minio --storage.enable-multicontainer --storage.container my-s3-bucket  --storage.connection.access-key minio --storage.connection.secret-key miniostorage 
INFO[0000] [0] Couldn't find a config file []. Relying on env vars and pflags. 
{"json":{},"level":"warning","msg":"stow configuration section missing, defaulting to legacy s3/minio connection config","ts":"2021-11-02T13:09:53+05:30"}
{"json":{},"level":"info","msg":"[0] Couldn't find a config file []. Relying on env vars and pflags.","ts":"2021-11-02T13:09:53+05:30"}
Couldn't find a config file.
Validated config file successfully.
```

Before the change stow configuration though not defined , it would be included in the config with empty kind attribute and wont take legacyS3ConfigMap to set the access key and secret key correctly and would fail.